### PR TITLE
T naabus/i frame csp compatible

### PR DESCRIPTION
**Pull request by t-naabus**

### CODE REVIEW

File: src/embed.ts [hunk 1]
```diff
@@ -467,9 +467,18 @@ export abstract class Embed {
    */
   private setIframe(isLoad: boolean): void {
     if(!this.iframe) {
-      const iframeHtml = `<iframe style="width:100%;height:100%;" src="${this.config.embedUrl}" scrolling="no" allowfullscreen="true"></iframe>`;
-      this.element.innerHTML = iframeHtml;
-      this.iframe = <HTMLIFrameElement>this.element.childNodes[0];
+      var iframeContent = document.createElement("IFRAME");
```
Comment (ArinaHantsis) [inline src/embed.ts]: Will all the created Iframes have the same identifier? How can we differentiate them? By the container div??

Comment (t-naabus) [inline src/embed.ts]: If I understood the question correctly, yes. the iframes are differentiated by the containers' id.

File: src/embed.ts [hunk 2]
```diff
@@ -467,9 +467,18 @@ export abstract class Embed {
...
+      var embedUrl = this.config.embedUrl;
+      iframeContent.setAttribute("style", "width:100%;height:100%;");
```
Comment (ali-hamud) [inline src/embed.ts]: instead of style, use class and declare the style in css file.

File: src/embed.ts [hunk 3]
```diff
@@ -467,9 +467,18 @@ export abstract class Embed {
...
+      iframeContent.setAttribute("src", embedUrl);
+      iframeContent.setAttribute("scrolling", "no");
+      iframeContent.setAttribute("allowfullscreen", "true");
+      var node = this.element;
+      while(node.firstChild) {
+          node.removeChild(node.firstChild);
+        }
```
Comment (ali-hamud) [inline src/embed.ts]: spaces

File: src/embed.ts (hunk 2 above)
Comment (t-naabus) [inline src/embed.ts]: We don't have a css file, css is present in the demo only.

**Pull request merged**